options/rtdl/generic/linker.cpp

```diff
@@ -22,6 +22,12 @@ constexpr bool logBaseAddresses = false;
 constexpr bool logRpath = false;
 constexpr bool eagerBinding = true;
 
+namespace {
+
+bool rpathWarned = false;
```

Comment:
Is this change really necessary?

Reply:
It massively spams the log when running some applications, so warning once seemed sufficient.

Reply:
Alright makes sense. I would prefer if it wasn't a global variable though, putting it as a static variable in the case scope should work. Also put a comment explaining that this is both temporary and to reduce log-spam for some applications.